app/assets/templates/home.html

```diff
@@ -3,7 +3,10 @@
 <div class="main-body container">
 
   <div class="jumbo row">
-    <img ng-repeat="recipe in recipes" class="splash-item img-responsive  col-xs-3 col-sm-2 col-lg-1" ng-src="{{ recipe.images[0] }}">
+    <div ng-repeat="recipe in recipes">
+      <img class="splash-item img-responsive  col-xs-3 col-sm-2 col-lg-1" ng-src="{{ recipe.images[0] }}">
```

Comment:
Can you explain where `recipe.images[0]` and `recipe.images[1]` come from? How come we have 2 different images?